src/__tests__/index-test.js

```diff
@@ -175,14 +179,22 @@ describe("toArray", () => {
     const content = toArray("https://google.com");
     expect(content).toMatchSnapshot();
   });
-  test("url and emoji (no space)", () => {
+  test("url and ascii emoji alias (no space)", () => {
     const content = toArray("https://google.com:)");
     expect(content).toMatchSnapshot();
   });
+  test("url and emoji (no space)", () => {
+    const content = toArray("https://google.com:heart:");
+    expect(content).toMatchSnapshot();
+  });
   test("url and emoji (with space)", () => {
```

Comment:
Let's rename this one so it has a different name than the one you added just below.

```suggestion
  test("url and ascii emoji (with space)", () => {
```